src/OrchardCore/OrchardCore.Abstractions/Json/JsonOptionsConfigurations.cs

```diff
@@ -14,8 +16,9 @@ public JsonOptionsConfigurations(IOptions<DocumentJsonSerializerOptions> jsonSer
         _jsonSerializerOptions = jsonSerializerOptions.Value.SerializerOptions;
     }
 
-    public void Configure(JsonOptions options)
-    {
-        options.SerializerOptions.Merge(_jsonSerializerOptions);
-    }
+    public void Configure(HttpJsonOptions options)
+        => options.SerializerOptions.Merge(_jsonSerializerOptions);
+
+    public void Configure(MvcJsonOptions options)
+        => options.JsonSerializerOptions.Merge(_jsonSerializerOptions);
```

Comment:
When does this get called for you? Because it doesn't for me.

Also, this would be better in two classes.

Reply:
> When does this get called for you? Because it doesn't for me.

Read the issue description

> Also, this would be better in two classes.

The problem with two classes is the naming

Reply:
We could use partial class for clarity

Reply:
I understand the issue description. My question is, when testing this, when do you see line 23 be called?

You can have two `IConfigureOptions` implementations, there's surely no difficulty in that.